charts/vmclarity/README.md

```diff
@@ -0,0 +1,68 @@
+# vmclarity
```

Comment:
please also add step in the pipeline that validates the charts and also makes sure readme is aligned with the changes